coordinatorlib/src/test/scala/com/socrata/datacoordinator/truth/loader/sql/StupidSqlLoader.scala

```diff
@@ -127,25 +154,18 @@ class StupidSqlLoader[CT, CV](val connection: Connection,
     }
   }
 
-  def findRow(id: CV): Option[InspectedRow[CV]] = {
-    using(sqlizer.findRows(connection, Iterator.single(id))) { blocks =>
-      val sids = blocks.flatten.toSeq
-      assert(sids.length < 2)
-      sids.headOption
-    }
-  }
-
-  def findRowId(id: CV): Option[InspectedRowless[CV]] = {
```

Comment:
it has long since been unused, since I would have to change it... I just wanted to delete it. Given I would be making changes to unused code that wouldn't be testable (without adding tests for it)